cypress/integration/1_new_engagement_spec.js

```diff
@@ -202,4 +202,10 @@ describe('new engagement', () => {
       'You have successfully launched your engagement!'
     );
   });
+  //This should be here, the alternate way to start an engagement
+  // it('can create new engagement by clicking the button', () => {
+  //   cy.visit('/app/engagements/all');
+  //   cy.get('[data-cy=create-new-engagement]').should('exist')
+  //   cy.get('[data-cy=create-new-engagement]').click();
+  // });
```

Comment:
I pulled this out. I would highly suggest one canonical way to Create an Engagement, that being the button in the Nav Drawer, even highlighted as a green button.

If we do decide to keep the Create Button on the filter bar, it should just repeat the test in Create. I'm not exactly sure how long this was broken.